Description: The correct option to obtain the list of valid variable names is
 --list-vars, not --list.
Author: Didier Raboud <odyx@debian.org>
Origin: vendor
Bugs: http://bugs.debian.org/681994
Last-Update: 2012-07-18

--- a/buildtools/config-script.in
+++ b/buildtools/config-script.in
@@ -15,7 +15,7 @@
     cat <<EOF
 usage: $0 --VAR ...
 
-Print the value of the variable VAR.  Run $0 -l or $0 --list for a list
+Print the value of the variable VAR.  Run $0 -l or $0 --list-vars for a list
 of valid variable names.
 EOF
 }
